.gitignore

```diff
@@ -1 +1,2 @@
 **/*.sw?
+/david_vicklund/node_modules/*
```

Comment:
since you are overwriting the share gitignore here, I'd suggest that you just create your own .gitignore in your folder and add the line 'node_modules/' to suppress just yours.